pages/blog/google-season-of-docs-2022.md

```diff
@@ -0,0 +1,102 @@
+
+![A screenshot displaying the design of our new feedback card for receiving feedback on AsyncAPI Docs](/img/posts/gsod-2022/feedback-card.webp)
+
+![A screenshot of AsyncAPI GitHub Discussions for the Docs category](/img/posts/gsod-2022/docs-community-discussions.webp)  
+
+The other way we would consider the project successful is the number of our contributors and Docs PRs increased by even 2-5 more community members. Currently, a majority of our OSS contributor community focuses only on contributing code, but we would like to instill a greater interest in contributing to documentation that provides value for everyone.
+
+
+### ⌛ Timeline
+The project itself will take approximately 3-6 months to complete, depending on how many technical writers (TW) get involved. If we can get even just 2-4 TWs, then we can work on both migrating the CLI/Tools documentation and the Generator Docs. But if we can only get 1-2 folks, it would be hard to do more than the migration of CLI and Tools docs. Ideally, the timeline could look like the following:
+
+- **May:** Orientation on how to contribute to AsyncAPI Inititiave, how Docs issues are organized, detail how we're migrating our CLI and Tools Docs, and assign good `first-time-tickets` to get each new TW contributor started. 
+- **June - August:**	Each TW goes through designated issues marked for both first time contributors and work set aside for `GSoD 2022`. Each TW starts creating documentation for their individual issues assigned/selected. 
+- **September - October:** We determine if we're going to be able to complete both CLI and Tools Docs plus the Generator Docs, depending on how many TWs are in our group and how much they've been able to complete so far. We re-align priorities as needed and asses what is missing to reach our 2022 IA change goals for AsyncAPI Docs.
+- **November:**	Project completion and all contributors receive some swag! 
+
+
+### 💸 Project budget
+We have set aside 2 mentors for now, for our 2 projects: improving our IA and re-structuring our Generator Docs. Should we be selected, AsyncAPI would like to request from Google a US $5000 budget for each project. For both projects, the request then totals for a $10,000 budget.  
```

Comment:
It would be great to have a table here with two/three columns: `Item`, `Budget` and `Justification`. Item would mention the exact cause the funds are going for, the budget can be $5000 while the justification can be the statement around what are the funds being assigned for.

Reply:
- Yes, a table is a nicer view for sure.

- We don't really need a `Justification` tho because we're only asking for money for the project TWs. And according to the Google template, that section for this item is blank: https://developers.google.com/season-of-docs/docs/org-proposal-template#sample_budget